src/index.html

```diff
@@ -16,7 +21,484 @@
+              href="tel:+12125357710"
+              class="navigation__phone"
+            >
+              +1 212-535-7710
+            </a>
+            <p class="navigation__label">The Met Cloisters</p>
+            <a
+              href="tel:+12129233700"
+              class="navigation__phone"
+            >
+              +1 212-923-3700
+            </a>
+          </div>
+        </div>
+        <a
+          href="#menu"
+          id="show-menu-btn"
+          class="navigation__icon navigation__icon--menu"
+        ></a>
+      </nav>
+      <h1 class="title">
+        Welcome
+        <br />
+        to The MET
+      </h1>
+    </header>
+    <div
+      class="menu"
+      id="menu"
+    >
+      <nav class="navigation menu__navigation">
+        <a
+          href="#"
+          class="navigation__logo-link"
+        >
+          <img
+            src="./images/icons/logo.svg"
+            alt="Logo The Met"
+          />
+        </a>
+        <a
+          href="#menu"
+          id="hide-menu-btn"
+          class="navigation__icon navigation__icon--close"
+        ></a>
+      </nav>
+      <nav class="page-navigation">
+        <ul class="page-navigation__list">
+          <li>
+            <a
+              href="#home"
+              class="page-navigation__link"
+            >
+              Home
+            </a>
+          </li>
+          <li>
+            <a
+              href="#information"
+              class="page-navigation__link"
+            >
+              About Us
+            </a>
+          </li>
+          <li>
+            <a
+              href="#exhibition"
+              class="page-navigation__link"
+            >
+              Now on view
+            </a>
+          </li>
+          <li>
+            <a
+              href="#gallery"
+              class="page-navigation__link"
+            >
+              Explore Online
+            </a>
+          </li>
+          <li>
+            <a
+              href="#contact"
+              class="page-navigation__link"
+            >
+              Contacts
+            </a>
+          </li>
+        </ul>
+        <p class="menu__label">The Met Fifth Avenue</p>
+        <a
+          href="tel:+12125357710"
+          class="button--secondary menu__phone"
+        >
+          +1 212-535-7710
+        </a>
+        <p class="menu__label">The Met Cloisters</p>
+        <a
+          href="tel:+12129233700"
+          class="button--secondary menu__phone"
+        >
+          +1 212-923-3700
+        </a>
+        <br />
+        <a
+          href="tel:+12125357710"
+          class="button--secondary"
+        >
+          call to order
+        </a>
+      </nav>
+    </div>
+
+    <main class="main">
+      <section
+        class="information main__section"
+        id="information"
+      >
+        <div>
+          <p class="title__description">Plan your visit</p>
+          <h2 class="title--secondary information__title">Museum hours</h2>
```

Comment:
if you want to use modifier then you should also use original block or element that the modifier is applied on. As i saw in styles there is some repeated code for `title` and `title--secondary`